_topic_maps/_topic_map.yml

```diff
@@ -1454,22 +1454,36 @@ Topics:
   Topics:
   - Name: Understanding multiple networks
     File: understanding-multiple-networks
-  - Name: Configuring an additional network
-    File: configuring-additional-network
+  - Name: Primary networks
+    Dir: primary_networks
+    Distros: openshift-enterprise, openshift-origin
+    Topics:
+    - Name: Using NetworkAttachmentDefinition
+      File: creating-primary-nad
+  - Name: Secondary networks
+    Dir: secondary_networks
+    Distros: openshift-enterprise, openshift-origin
+    Topics:
+    - Name: Creating secondary networks on OVN-Kubernetes network plugin
```

Comment:
Checking on this but I feel like I have to use network plugin. I didn't work on that language though when we did it awhile back so I'll check and see. For consistency, I _think_ because this doesn't come under the OVN-K network plugin section in the docs we might have to. 🤷🏻

Reply:
I got rid of this.